src/events/reaction-add.ts

```diff
@@ -33,27 +37,15 @@ const addRolesFromReaction = async (
     },
   });
 
+  const guildMember =
+    guild.member(user.id) ?? (await guild.members.fetch(user.id));
+  //Since most of the bot isn't refactored yet, this must stay for the old and new event do not collide together. Color Roles are handled in Nitro-colors.ts
+  if (isColorSelectionMessage(messageId) && memberHasNitroColor(guildMember))
```

Comment:
This still suffered from some weird race-condition. This should fix it though (we don't want this file to handle nitro colors, no matter whether the member already has a nitro color or not):
```suggestion
  if (isColorSelectionMessage(messageId))
```